### PR TITLE
DACCESS 4 Sprint - Merge dev to main

### PR DESCRIPTION
**Pull request by chrisrlc**
Production deploy is planned for tomorrow, Tuesday 12/10, preferably morning!

### CODE REVIEW

**Review by mhk33 — APPROVED** [review]

**Review by sk274 — APPROVED** [review]

**Review by JeremyDuncan — APPROVED** [review]

File: blacklight-cornell/app/controllers/catalog_controller.rb
Comment (jgreidy) [inline blacklight-cornell/app/controllers/catalog_controller.rb:1]: Nice reorganization.

File: blacklight-cornell/app/models/concerns/blacklight/solr/document/marc_export.rb
Comment (jgreidy) [inline blacklight-cornell/app/models/concerns/blacklight/solr/document/marc_export.rb:1]: Bye bye Cite!

File: blacklight-cornell/app/models/search_builder.rb
Comment (jgreidy) [inline blacklight-cornell/app/models/search_builder.rb:1]: Great. I'm glad my editor's "whitespace" filter did not get a hold of this one!

**Review by sarah-cul — APPROVED** [review]

File: blacklight-cornell/spec/models/search_builder_spec.rb
Comment (jgreidy) [inline blacklight-cornell/spec/models/search_builder_spec.rb:1]: Wow

**Review by jgreidy — APPROVED** [review]
Nice work everybody!

File: blacklight-cornell/features/catalog_search/book_bags.feature [hunk 1]
```diff
@@ -110,7 +110,7 @@ Feature: Book Bags for logged in users
 
 
     @book_bags_cite_selected
-    Scenario Outline: I should be able to view citations for selected items
+    Scenario: I should NOT be able to view citations for selected items
```
Comment (Baroquem) [inline blacklight-cornell/features/catalog_search/book_bags.feature:113]: I get the intent here, but I don't think we should keep tests like this in place long-term. It takes so long to run the entire test suite now that we probably want to eliminate any tests that we don't need. Weigh that against the risk of regression bugs, I suppose.

**Review by Baroquem — APPROVED** [review]

**Pull request merged**